### ISSUE TITLE
Preprocess Metadata Step Not Getting Data

### ISSUE THREAD

**Issue by roshankern** [cheeseman-lab/brieflow#14]
The `extract_metadata_sbs` and `extract_metadata_phenotype` steps do not correctly extract metadata. When running these steps, my console has the following errors for all ND2 files:

```
Error processing data/input/sbs/P001_SBS_10x_C4_Wells-A1_Points-001__Channel_Cy7,Cy5,AF594,Cy3_SBS,DAPI_SBS.nd2: list index out of range
Error processing data/input/sbs/P001_SBS_10x_C4_Wells-A1_Points-100__Channel_Cy7,Cy5,AF594,Cy3_SBS,DAPI_SBS.nd2: list index out of range
No valid ND2 files found in the provided list.
Error processing data/input/sbs/P001_SBS_10x_C10_Wells-A1_Points-001__Channel_Cy7,Cy5,AF594,Cy3_SBS,DAPI_SBS.nd2: list index out of range
Error processing data/input/sbs/P001_SBS_10x_C10_Wells-A1_Points-100__Channel_Cy7,Cy5,AF594,Cy3_SBS,DAPI_SBS.nd2: list index out of range
No valid ND2 files found in the provided list.
```

The metadata tsvs that result from these rules (ex `example_analysis/analysis_root/preprocess/metadata/sbs/WA1_C1__metadata.tsv`) are completely empty.

This seems to be a result of #9 as reverting these changes solves the issue.

@mat10d are you able to reproduce this issue on your machine? Maybe this is a problem specific to my setup.

**Comment by mat10d** [comment]
Check if [this](https://github.com/cheeseman-lab/brieflow/pull/15) resolves this, and close if it does. Sorry for the headache.

**Comment by roshankern** [comment]
Fixed in #15 !
